components/list/list-item-mixin.js

```diff
@@ -81,10 +82,20 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 				border-bottom: 1px solid transparent;
 				border-top: 1px solid transparent;
 			}
-			[slot="content"].d2l-list-item-content-extend-separators {
```

Comment:
This selector changed because the `d2l-list-item-content-extend-separators` class was moved up to the parent, so that it could be used to select both the `content` and `control` slots when separators are being extended.